### PR TITLE
Importing Data feature

### PR DESCRIPTION
**Pull request by zephyllr**
Created Script class with attributes theme and path. 
Diff paths necessary to access scripts for testing and running server. 
Content is pulled from scripts/ directory. 
scripts/ dir contains a file for the several movie scripts (for an excerpt scene) and test script called Sample.txt. 
AdminServlet's doPost controls import data functionality.
Test created for Scripts.java. Will create one for AdminServlet.java. 
MVP completed (yay), though not all edge cases are taken care of at the moment.

### CODE REVIEW

File: scripts/WallE.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+EVE: Wally.
+EVE: Eve.
+WALLY: Eeee...?
+EVE: Eve.
+WALLY: Eeeaaah?
+EVE: Eeeve. Eeeve.
+WALLY: Eee--vah!
+WALLY: Eee-vah! Ee --
+EVE: Eve.
+WALLY: Eee-vah!
```
Comment (tatounel) [inline scripts/WallE.txt]: Lol I love this!

File: src/main/java/codeu/controller/AdminServlet.java [hunk 2]
```diff
@@ -41,11 +70,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
...
 
-    if (script.equals("wall_e")) {
-    	System.out.println("WAALLLY");
-    }
+    String theme = request.getParameter("script");
+    Script script = new Script(theme);
+    System.out.println(script.getTheme());
+    System.out.println(script.getContent());
+
+    // for
```
Comment (tatounel) [inline src/main/java/codeu/controller/AdminServlet.java]: remove comment?

File: src/main/java/codeu/model/data/Script.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  }
+
+  /** Returns the theme of this Script. */
+  public String getTheme() {
+    return theme;
+  }
+
+  /** Returns the content of this Script. */
+  public Map<String, List<String>> getContent() {
+    System.out.println("GOT CONTENT!!");
```
Comment (tatounel) [inline src/main/java/codeu/model/data/Script.java]: do you still want to print this?

File: src/main/java/codeu/model/data/Script.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    return content;
+  }
+
+  public Map<String, List<String>> mapScript(){
+    System.out.println("MAPPING!!");
```
Comment (tatounel) [inline src/main/java/codeu/model/data/Script.java]: don't know if you still want to print this

File: src/test/java/codeu/model/data/ScriptTest.java [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+public class ScriptTest {
+
+  // private UserStore userStore;
+  // private PersistentStorageAgent mockPersistentStorageAgent;
+
+  // @Before
+  // public void setup() {
+  //   mockPersistentStorageAgent = Mockito.mock(PersistentStorageAgent.class);
+  //   userStore = UserStore.getTestInstance(mockPersistentStorageAgent);
+  // }
```
Comment (tatounel) [inline src/test/java/codeu/model/data/ScriptTest.java]: remove comments?

**Review by tatounel — APPROVED** [review]
Great job!
